Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -2132,12 +2132,15 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     {
         if (row == CRYPTOGRAPHY_INFO_INDEX)
         {
-            MXKTableViewCell *cryptoCell = [self getDefaultTableViewCell:tableView];
-
-            cryptoCell.textLabel.attributedText = [self cryptographyInformation];
-            cryptoCell.textLabel.numberOfLines = 0;
+            MXKTableViewCellWithTextView *cryptoCell = [tableView dequeueReusableCellWithIdentifier:[MXKTableViewCellWithTextView defaultReuseIdentifier]];
```

Comment:
Could you keep using `getDefaultTableViewCell`?
It manages theme for you. With the current code in this PR, you get this display in dark theme:
![simulator screen shot - iphone 6 - 2018-08-24 at 10 51 42](https://user-images.githubusercontent.com/8418515/44575526-81d0b980-a78c-11e8-8f1d-89a5ad54a5ba.png)